projects/1.20/assets/pams-harvestcraft-2-food-core/pamhc2foodcore/lang/zh_cn.json

```
@@ -0,0 +1,184 @@
+{
+  "item.pamhc2foodcore.applejellyitem": "苹果酱",
+  "item.pamhc2foodcore.applejellytoastitem": "苹果酱吐司",
+  "item.pamhc2foodcore.applejuiceitem": "苹果汁",
+  "item.pamhc2foodcore.applepieitem": "苹果派",
+  "item.pamhc2foodcore.applemuffinitem": "苹果松饼",
+  "item.pamhc2foodcore.applesauceitem": "苹果沙司",
+  "item.pamhc2foodcore.applesmoothieitem": "苹果沙冰",
+  "item.pamhc2foodcore.appleyogurtitem": "苹果酸奶",
+  "item.pamhc2foodcore.baconandeggsitem": "熏肉蛋",
+  "item.pamhc2foodcore.baconcheeseburgeritem": "培根芝士堡",
+  "item.pamhc2foodcore.bakedvegetablemedlyitem": "烤蔬菜杂烩",
+  "item.pamhc2foodcore.bakewareitem": "烤盘",
+  "item.pamhc2foodcore.basiccheeseburgeritem": "奶酪牛肉堡",
+  "item.pamhc2foodcore.basicchickensandwichitem": "鸡肉堡",
+  "item.pamhc2foodcore.basicfishsandwichitem": "鱼肉堡",
+  "item.pamhc2foodcore.basichamburgeritem": "牛肉堡",
+  "item.pamhc2foodcore.batteritem": "面糊",
+  "item.pamhc2foodcore.beefjerkyitem": "牛肉干",
+  "item.pamhc2foodcore.beefpotpieitem": "牛肉派",
+  "item.pamhc2foodcore.boiledeggitem": "煮鸡蛋",
+  "item.pamhc2foodcore.butteredbakedpotatoitem": "黄油烤马铃薯",
+  "item.pamhc2foodcore.butteritem": "黄油",
+  "item.pamhc2foodcore.caramelappleitem": "焦糖苹果",
+  "item.pamhc2foodcore.caramelicecreamitem": "焦糖冰激凌",
```

```suggestion
  "item.pamhc2foodcore.caramelicecreamitem": "焦糖冰淇淋",
```
#3935 
https://www.termonline.cn/search?searchText=ice+cream